lib/fluent/plugin/in_syslog.rb

```diff
@@ -199,14 +199,22 @@ def start_tcp_server(tls: false)
 
     private
 
+    def emit_unmatched(data, sock)
+      record = {}
```

Comment:
`record = {"unmatched_line" => data}` is better. No need line 206

Reply:
Yip. Changed and force pushed.